tasks/pools.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+- name: Setup rsyslog pool configuration
+  template:
+    src: 'etc/rsyslog.d/pool.conf.j2'
+    dest: '/etc/rsyslog.d/{{ item.name }}.conf'
+    owner: '{{ item.owner | default("root") }}'
+    group: '{{ item.group | default("root") }}'
+    mode: '{{ item.mode | default("0644") }}'
+    backup: '{{ item.backup | default("yes")  }}'
```

Comment:
I would switch the default to `no`, I'm not sure if `rsyslog` ignores files with `~` in their filename? Anyway, having default `no` here will let you omit `item.backup` key everywhere but where you want backup to be actually made.